orchagent/fdborch.cpp

```diff
@@ -73,30 +74,39 @@ bool FdbOrch::storeFdbEntryState(const FdbUpdate& update)
 
     if (update.add)
     {
-        auto inserted = m_entries.insert(entry);
-
-        SWSS_LOG_DEBUG("FdbOrch notification: mac %s was inserted into bv_id 0x%" PRIx64,
-                        entry.mac.to_string().c_str(), entry.bv_id);
-
-        if (!inserted.second)
+        bool mac_move=false;
+        auto it = m_entries.find(entry);
+        if(it != m_entries.end())
         {
-            SWSS_LOG_INFO("FdbOrch notification: mac %s is duplicate", entry.mac.to_string().c_str());
-            return false;
+            if(port.m_bridge_port_id == it->second.bridge_port_id)
```

Comment:
In theory, MAC type may also be judged, even though type changing may not be notified by syncd.

Reply:
Type change from static to dynamic is not expected and should not be sent from HW/SAI, as it will require delete and re-creating of the fdb entry, as static fdb has more preference.

Reply:
In current implementation, fdb type is not notified by HW/SAI, thus the default type is dynamic, this may be fixed in future.

Reply:
I am not sure in what scenario SAI will notify about a learn event with type as static. static mac is usually added from sonic down to SAI.

Reply:
Yes, current implementation is like this.